gradle/shipkit.gradle

```diff
@@ -17,28 +16,45 @@ tasks.named("githubRelease") {
     githubToken = System.getenv("GITHUB_TOKEN")
 }
 
-/**
- * Mockito team does not publish to Maven Central every release as requested by the community.
- * We publish to maven central only some versions. All versions are published to Bintray on every change.
- * See https://github.com/mockito/mockito/issues/911
- */
-ext.mavenCentralRelease = shouldReleaseToCentral(project)
+apply plugin: 'io.github.gradle-nexus.publish-plugin'
+nexusPublishing {
+    repositories {
+        if (System.getenv('NEXUS_TOKEN_PWD')) {
+            sonatype {
+                // Publishing to: https://s01.oss.sonatype.org (faster instance)
+                nexusUrl = uri('https://s01.oss.sonatype.org/service/local/')
+                snapshotRepositoryUrl = uri('https://s01.oss.sonatype.org/content/repositories/snapshots/')
 
-/**
- * Finds out if we should release to Maven Central.
- * To test this logic, run build with '-i' (info logging) and inspect the build output.
- */
-static boolean shouldReleaseToCentral(project) {
-    boolean centralReleaseByCommit = 'true' == System.getenv("MAVEN_CENTRAL_RELEASE")
-    boolean centralReleaseByProjectProperty = project.hasProperty("maven-central-release")
-    boolean centralRelease = centralReleaseByCommit || centralReleaseByProjectProperty
-    def message = """Bintray publish was using following settings:
- - commit message contains '[ci maven-central-release]': $centralReleaseByCommit
- - project property 'maven-central-release' exists: $centralReleaseByProjectProperty
- - Maven Central release is enabled: $centralRelease"""
-    project.logger.info(message)
-    project.afterEvaluate {
-        project.tasks.bintrayPublish.doLast { logger.lifecycle(message) }
+                username = System.getenv('NEXUS_TOKEN_USER')
+                password = System.getenv('NEXUS_TOKEN_PWD')
```

Comment:
Cool that you use the [user token](https://blog.solidsoft.pl/2015/09/08/deploy-to-maven-central-using-api-key-aka-auth-token/) instead of the regular credentials!